.githooks/pre-commit

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+SRC_PATTERN="guides/release/"
+changes=`git diff --cached --name-only | grep "$SRC_PATTERN"`
+for translated in $changes; do
+  while read -r line; do
+    if [ "!$translated" = "$line" ]; then
+      exit 0
```

Comment:
This kill the process if the first file is in the remarkignore, but if you have many files it won't check the other files

Reply:
Yes, for the V1 it's fine,  we normally translate only one file per MR, so there's no reason to have several `.md` modified.